runtime/src/bank.rs

```diff
@@ -4258,6 +4259,17 @@ pub fn goto_end_of_slot(bank: &mut Bank) {
     }
 }
 
+use solana_vote_program::vote_state::BlockTimestamp;
+use solana_vote_program::vote_state::VoteStateVersions;
+pub fn update_vote_account_timestamp(timestamp: BlockTimestamp, bank: &Bank, vote_pubkey: &Pubkey) {
```

Comment:
i think we can move this into the test module below (make the module public for the crate), and then from  tests in `bank_forks` that need the import, import `bank::tests::update_vote_account_timestamp,`

Reply:
https://github.com/solana-labs/solana/pull/13477/commits/1721cb69af4c9f50fa98d638a7a9a15883914395